### PR TITLE
Added AddGloveboxItems Event & added support job.type

### PR DESCRIPTION
**Pull request by r0adra93**
Added addGloveboxItems event
Added job.type leo match up to the script.

## Description

Added AddGloveboxItems Event & added support job.type

Now glove box items can be added from jobs
and the job.type is setup to work with leo type of police jobs.

## Checklist

- [X] I have personally loaded this code into an updated qbcore project and checked all of its functionality.
- [X] My code fits the style guidelines.
- [X] My PR fits the contribution guidelines.

### CODE REVIEW

**Review by i-kulgu — APPROVED** [review]
Tested it out and works without any issues.

File: server/main.lua [hunk 1]
```diff
@@ -1346,6 +1349,7 @@ RegisterNetEvent('inventory:server:OpenInventory', function(name, id, other)
 					end
 				end
 			elseif name == "shop" then
+				QBCore.Debug(other.label)
```
Comment (tom-osborne) [inline server/main.lua:1352]: Debug left over from testing.

**Review by tom-osborne — CHANGES_REQUESTED** [review]

File: server/main.lua (hunk 1 above)
Comment (tom-osborne) [reply server/main.lua:1352]: TMI

Thanks though!

Comment (r0adra93) [reply server/main.lua]: Updated!

**Review by tom-osborne — APPROVED** [review]

**Pull request merged**